src/main/java/core/basesyntax/service/impl/DataParseServiceImpl.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax.service.impl;
+
+import static java.lang.Integer.parseInt;
+
+import core.basesyntax.model.StorageTransaction;
+import core.basesyntax.service.DataParseService;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class DataParseServiceImpl implements DataParseService {
+    public static final int ACTIVITY_INDEX = 0;
+    public static final int FRUIT_INDEX = 1;
+    public static final int QUANTITY_INDEX = 2;
+    public static final String PATTERN = ",";
+
+    @Override
+    public List<StorageTransaction> getParsedData(List<String> data) {
+        return data.stream()
+                .map(d -> d.split(PATTERN))
```

Comment:
do not use abbreviated names, 
fix everywhere
```suggestion
                .map(line -> line.split(PATTERN))
```